var/spack/repos/builtin/packages/nettle/package.py

```diff
@@ -22,3 +22,6 @@ class Nettle(AutotoolsPackage):
 
     depends_on('gmp')
     depends_on('m4', type='build')
+
+    def configure_args(self):
+        return ['CFLAGS=-std=c99']
```

Comment:
We should update this according to #11618 (thanks for that!):
```suggestion
        return ['CFLAGS={0}'.format(self.compiler.c99_flag)]
```